.github/workflows/generate_binary_build_matrix.yml

```diff
@@ -35,6 +35,10 @@ on:
         description: "Build with CPU?"
         default: "enable"
         type: string
+      with-xpu:
+        description: "Build with XPU?"
+        default: "enable"
```

Comment:
Please keep it disable for now. Since this will automatically enable vision and audio builds. I don't think we want this at this point.

Reply:
Hi @atalman thanks for the review, we plan to enable vision and audio build also for now. So that we can have completed nightly install cmd like `pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/xpu`. How do you think? Any concern in here?

Reply:
hi @chuanqi129 please set default to disable since multiple domains use this script. Here are some examples:
https://github.com/pytorch/data/blob/main/.github/workflows/build_wheels_linux.yml#L29
https://github.com/pytorch/torchtune/blob/main/.github/workflows/build_linux_wheels.yaml#L29

Reply:
Oh, thanks @atalman for the reminder. Got your point now, I will set the `with_xpu` disable as default and enable it in torchvision and torchaudio after this PR land, does it make sense to you?

Reply:
changed